fvm/src/call_manager/default.rs

```diff
@@ -595,7 +595,12 @@ where
                 // If the invocation failed due to running out of exec_units, we have already
                 // detected it and returned OutOfGas above. Any other invocation failure is returned
                 // here as an Abort
-                Ok(res?)
+
+                Ok(res.map_err(|e: anyhow::Error|match e.anyhow_kind() {
+                    // translate  anyhow error we get into an abort
+                    wasmtime::Trap => e,
+                    Abort => e,
+                })?)
```

Comment:
This is trying to translate from the anyhow error we get from wasm call into an abort, tho I'm not quite able to find a good way to do that

Reply:
Ah, you need `downcast`.